2025-07-30 04:27:39,930 ERROR import_views Error importing book
Traceback (most recent call last):
  File "/home/runner/work/lexiflux/lexiflux/lexiflux/views/import_views.py", line 49, in import_book
    raise ValueError(f"Unknown import type: {import_type}")
ValueError: Unknown import type: unknown_type
